documentation-site/components/yard/ast.ts

```diff
@@ -1,17 +1,25 @@
-import prettier from 'prettier/standalone';
 import traverse from '@babel/traverse';
 import generate from '@babel/generator';
+import {formatCode} from './code-generator';
 import * as t from 'babel-types';
-import babel from 'prettier/parser-babylon';
+import {TProp} from './types';
+import {PropTypes} from './const';
+import {parse as babelParse} from '@babel/parser';
 
-const parse = babel.parsers.babel.parse as (code: string) => any;
+export const parse = (code: string) =>
+  babelParse(code, {
+    sourceType: 'module',
+    plugins: ['jsx'],
+  });
 
 // clean-up for react-live, removing all imports, exports and top level
-// variable declaration
-export const removeImportsAndExports = (code: string) => {
-  let result = code;
+// variable declaration, add __yard_onChange instrumentation when needed
```

Comment:
Oh good catch. It should actually strip that statement completely. There needs to be a single expression (function component or class component) since it's evaled as

```js
`return ${resultCode}`
```

will fix

Reply:
actually, it's correct.. the purpose of this is to change

```jsx
const Component = () => <div />;
```

into

```jsx
() => <div />;
```

not really to detect/remove other top level variables